3.2/Dockerfile

```diff
@@ -1,5 +1,5 @@
 FROM ruby:3.2
-MAINTAINER Codeminer42 <contact@codeminer42.com>
+LABEL org.opencontainers.image.authors="contact@codeminer42.com"
```

Comment:
This change is just to remove a deprecation warning 💅 

https://docs.docker.com/reference/build-checks/maintainer-deprecated/